src/server/server.ts

```diff
@@ -0,0 +1,16 @@
+import { createConnection } from "typeorm";
+import { ApolloServer } from "apollo-server";
+import { MerchantResolver, CartResolver } from "./modules/merchant/resolvers";
+import { buildSchema } from "type-graphql";
+
+createConnection().then(async () => {
+  const schema = await buildSchema({
+    resolvers: [MerchantResolver, CartResolver],
+    validate: false
+  })
+
+  const server = new ApolloServer({ schema, introspection: true, playground: true })
```

Comment:
enabled graphiql and introspection for debugging purposes, preferably turned off for production